runtime/util/romclasswalk.c

```diff
@@ -514,12 +514,16 @@ static void allSlotsInBytecodesDo(J9ROMClass* romClass, J9ROMMethod* method, J9R
 			case JBputstatic:
 			case JBgetfield:
 			case JBputfield:
+			case JBwithfield:
```

Comment:
Is there a reason why this and `JBdefaultvalue` below are not in `#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)` ?

Reply:
No, I only did it this way because they are not wrapped in ifdef in `RomClassWalker.java` as well, so I didn't notice.
Should I put the 2 of them together in one `ifdef` or preserve the current order and have 2 `ifdefs`?

Reply:
The current order looks reasonable (`JBwithfield` next to `JBputfield` and `JBdefaultvalue` next to `JBnew`). You can put them in 2 `ifdefs`.

Reply:
Done